src/Notifications/VerificationCodeInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace NextApps\VerificationCode\Notifications;
+
+interface VerificationCodeInterface
```

Comment:
should we not better name this `VerificationCodeCreatedInterface`, similar to the real notification class `VerificationCodeCreated`? 
`VerificationCodeInterface` seems like you can override the `VerificationCode` model class